x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/IntArrayVector.java

```diff
@@ -92,6 +102,36 @@ public static long ramBytesEstimated(int[] values) {
         return BASE_RAM_BYTES_USED + RamUsageEstimator.sizeOf(values);
     }
 
+    /**
+     * The minimum value in the block.
+     */
+    @Override
+    public int min() {
+        if (min == null) {
+            int v = values[0];
```

Comment:
I think we can create an empty array vector?

Reply:
Can we!? I'll check that.

Reply:
Looks like you can make one. I've made min into `Integer.MAX_VALUE` if you do it on an empty one.